### PR TITLE
[test](mtmv)add create partition and partition scrolling

### PR DESCRIPTION
**Pull request by zfr9527**
## Proposed changes

[test](mtmv)add create partition and partition scrolling

### CODE REVIEW

**Comment by zfr9527** [comment]
run buildall

**Review by morrySnow — DISMISSED** [review]

File: regression-test/suites/nereids_rules_p0/mv/union_rewrite/partition_curd_union_rewrite_hive.groovy [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+
+
+        // Test when base table create partition
+        sql """
+            insert into ${catalog_name}.${db}.${lineitem_tb_name} values 
+            (1, 2, 3, 4, 5.5, 6.5, 7.5, 8.5, 'o', 'k', '2023-10-21', '2023-10-21', 'a', 'b', 'yyyyyyyyy', '2023-10-21');
+            """
+        // Wait partition is invalid
+        sleep(5000)
+        explain {
```
Comment (seawinde) [inline regression-test/suites/nereids_rules_p0/mv/union_rewrite/partition_curd_union_rewrite_hive.groovy:261]: Should use `mv_rewrite_success` in Suite.grovvy, to location not chosen mv reason easier

Comment (zfr9527) [reply regression-test/suites/nereids_rules_p0/mv/union_rewrite/partition_curd_union_rewrite_hive.groovy:261]: All have been modified completely.

**Comment by zfr9527** [comment]
run buildall

**Review by seawinde — APPROVED** [review]

**Review by morrySnow — APPROVED** [review]

**Pull request merged**